CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
+- Added missing `console` polyfills in release builds. ([b7ab922](https://github.com/facebook/react-native/commit/b7ab922) by [@yungsters](https://github.com/yungsters))
+- Add missing accessibility props on Touchables ([8c0c860](https://github.com/facebook/react-native/commit/8c0c860) by [@xuelgong](https://github.com/xuelgong))
+- Changelog: [General][added] - Add getNativeScrollRef method to FlatList component ([bde1d63](https://github.com/facebook/react-native/commit/bde1d63))
+- Clean up unused functions on ReactContext ([3963c7a](https://github.com/facebook/react-native/commit/3963c7a) by [@ejanzer](https://github.com/ejanzer))
+- Add test for the `processColorArray` to make sure it maps correctly ([3c4b1d6](https://github.com/facebook/react-native/commit/3c4b1d6) by [@talhaazhar](https://github.com/talhaazhar))
+
+#### Android specific
+
+- Give ReactContextBaseJavaModule a 0 arg ctor ([e69be0a](https://github.com/facebook/react-native/commit/e69be0a) by [@RSNara](https://github.com/RSNara))
+- Bump gradle-download-task to 4.0.2 ([088be26](https://github.com/facebook/react-native/commit/088be26) by [@dulmandakh](https://github.com/dulmandakh))
+- Implement TurboModule eager initialization support ([b7f49a8](https://github.com/facebook/react-native/commit/b7f49a8) by [@RSNara](https://github.com/RSNara))
+
+#### iOS specific
+
+- Summary: Changelog: [iOS][changed] - Added another build flag for DevLoadingView ([3ceb959](https://github.com/facebook/react-native/commit/3ceb959) by [@mmmulani](https://github.com/mmmulani))
+- Make RCTDevLoadingView TurboModule-compatible ([294e31b](https://github.com/facebook/react-native/commit/294e31b) by [@RSNara](https://github.com/RSNara))
+- Fix Flipper integration on and update Flipper to 0.30.0 ([898b1db](https://github.com/facebook/react-native/commit/898b1db) by [@charpeni](https://github.com/charpeni))
+
+### Changed
+
+- [General][changed] TextInput now uses `forwardRef` allowing it to be used directly by new APIs requiring a host component. ([bbc5c35](https://github.com/facebook/react-native/commit/bbc5c35) by [@TheSavior](https://github.com/TheSavior))
+- Changelog: [General][changed] - React sync for revisions 0b61e2698...6cff70a74 (Includes React 16.11.0) ([1edce56](https://github.com/facebook/react-native/commit/1edce56) by [@gaearon](https://github.com/gaearon))
+- Don't crash when pretty-format is not available ([25a21b8](https://github.com/facebook/react-native/commit/25a21b8))
+- [General][changed] Partial React Sync from 6cff70a74...bb1853f73 ([222a089](https://github.com/facebook/react-native/commit/222a089) by [@TheSavior](https://github.com/TheSavior))
+- [General][fixed] TextInput now properly sends native the end selection location on change ([dff490d](https://github.com/facebook/react-native/commit/dff490d) by [@TheSavior](https://github.com/TheSavior))
```

Comment:
I did something weird on this commit. I couldn't break it up into separate commits so I included two changelog lines because I kinda wanted two different line items here. Can you add another line item to the same commit with this message?

```
[General][Changed] Converted TextInput to use React hooks
```